### ISSUE TITLE
VPAT — Poor Contrast Ratio In New Gradeables View

### ISSUE THREAD

**Issue by pbailie** [Submitty/Submitty#4836]
![Gradeables_view(2)](https://user-images.githubusercontent.com/12703624/70179351-e5de6f00-16ab-11ea-8087-3deb105f981f.png)

Grey text (disabled tab labels near top) that is circled in blue is colored `#888888` and has a contrast ratio of 3.54:1.  Contrast ratio should be at least 4.5:1.
